libweb3jsonrpc/UnixSocketServer.cpp

```diff
@@ -19,7 +19,7 @@ along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 *   Arkadiy Paronyan <arkadiy@ethdev.com>
 * @date 2015
 */
-#ifndef WIN32
+#if !defined(_WIN32)
```

Comment:
When checking compiler and build flags, I'm suggesting using `#if _WIN32`. When not defined the macro is substituted by 0.